### DISCUSSION TITLE
How do I configure Redis in Loco?

### DISCUSSION THREAD

**Discussion by quentin996007** [loco-rs/loco#798] [Q&A]
I used docker to start redis
'docker run -p 4333:6379 --name miser-redis -d redis --requirepass “abc”'
How do I configure Redis in Loco? And how do I access and manipulate Redis when I'm in the controller?
Please give me some guidance, thanks!

**Comment by jondot** [comment]
hi,
are you looking to use Redis as a cache, or as a queue provider?

**Reply by quentin996007** [reply]
Both are needed. The documentation doesn't describe this part, and I'm kind of stumped to get started. It would be nice to have simple examples.

**Reply by jondot** [reply]
For cache use the cache subsystem:
https://loco.rs/docs/infrastructure/cache/

For queue use cases see the worker docs:
https://loco.rs/docs/processing/workers/
